.eslintrc.json

```diff
@@ -0,0 +1,162 @@
+            "error",
+            "always"
+        ],
+        "padding-line-between-statements": [
+            "error",
+            {
+                "blankLine": "always",
+                "prev": "multiline-expression",
+                "next": "*"
+            },
+            {
+                "blankLine": "always",
+                "prev": "*",
+                "next": "multiline-expression"
+            },
+            {
+                "blankLine": "always",
+                "prev": "multiline-block-like",
+                "next": "*"
+            },
+            {
+                "blankLine": "always",
+                "prev": "*",
+                "next": "multiline-block-like"
+            },
+            {
+                "blankLine": "always",
+                "prev": "multiline-const",
+                "next": "*"
+            },
+            {
+                "blankLine": "always",
+                "prev": "*",
+                "next": "multiline-const"
+            },
+            {
+                "blankLine": "always",
+                "prev": "multiline-let",
+                "next": "*"
+            },
+            {
+                "blankLine": "always",
+                "prev": "*",
+                "next": "multiline-let"
+            },
+            {
+                "blankLine": "any",
+                "prev": "case",
+                "next": "case"
+            }
+        ],
+        "import/extensions": [
+            "error",
+            "ignorePackages"
+        ],
+        "import/no-unresolved": "off",
+        "import/no-cycle": [
+            "error"
+        ],
+        "import/order": [
+            "error",
+            {
+                "newlines-between": "always",
+                "alphabetize": {
+                    "order": "asc",
+                    "caseInsensitive": true
+                }
+            }
+        ],
+        "node/no-unsupported-features/es-syntax": "off",
+        "node/no-unpublished-import": "off",
+        "node/no-missing-import": "off",
+        "@typescript-eslint/explicit-member-accessibility": [
+            "error",
+            {
+                "accessibility": "no-public"
+            }
+        ],
+        "@typescript-eslint/strict-boolean-expressions": [
+            "error",
+            {
+                "allowString": false,
+                "allowNumber": false,
+                "allowNullableObject": false,
+                "allowNullableBoolean": false,
+                "allowNullableString": false,
+                "allowNullableNumber": false,
+                "allowAny": false
+            }
+        ],
+        "@typescript-eslint/consistent-type-assertions": [
+            "error",
+            {
+                "assertionStyle": "never"
+            }
+        ],
+        "unused-imports/no-unused-imports": "error",
+        "license-header/header": [
+            "error",
+            "./resources/license-header.js"
+        ]
+    }
+}
```

Comment:
Interesting that you still don't have new lines at the ends of the files. Check that prettier is running everywhere especially in CI except the places that you don't want it to run (you check out .prettierignore in CLI repo)

Reply:
Yeap, will refactor this a bit